torchvision/datasets/places365.py

```diff
@@ -13,7 +13,7 @@ class Places365(VisionDataset):
 
     Args:
         root (string): Root directory of the Places365 dataset.
-        split (string, optional): The dataset split. Can be one of ``train-standard`` (default), ``train-challendge``,
+        split (string, optional): The dataset split. Can be one of ``train-standard`` (default), ``train-challenge``,
```

Comment:
Good catch, but lets keep the scope of the PR narrow. Please send a follow-up PR with that change.